src/flags.ts

```diff
@@ -128,6 +128,16 @@ const expires = flags.build({
     "Specify a longer expiration date for public diffs (defaults to 1 day). Use iso8601 format to provide a date, or you can use `--expires 'never'` to keep the result live indefinitely.",
 });
 
+const out = flags.build({
+  char: 'o',
+  description: 'Output file path',
+});
+
+const overlay = flags.build({
+  char: 'o',
+  description: 'Path or URL of an overlay file to apply before deploying',
+});
```

Comment:
Do we really want to use the `o` char for both flags? (Real question, I'm OK with that if you think it cannot be misleading)

Reply:
I thought about it haha. In the `deploy` command I don't think we'll ever have an “output” so I think it's fine. What other short name would you recommend for this new `--overlay` flag if we don't use `-o`?

Reply:
What's the common practice in CLI tools in this kind of situation? I think we may use another letter that does appear in the word, couldn't we? Maybe a `-l`?

Zero strong opinion here: as the `deploy` command doesn't have an output with a definition, we can also keep it as is without any risk.

I let you take the decision on this one!

Reply:
I think I prefer to keep `-o`